google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -6,7 +6,7 @@ language_settings:
   python:
     package_name: google.pubsub.v1
   go:
-    package_name: google.golang.org/cloud/pubsub
+    package_name: cloud.google.com/go/pubsub/apiv1
```

Comment:
@vchudnov-g I just got confirmation from Dave Day. At least for now, `apiv1` -> VKit and `v1` -> Veneer, so that both can exist without stepping on the other's toes.